### ISSUE TITLE
Timeline for changing old testnet network

### ISSUE THREAD

**Issue by eric248550** [Berry-Pool/nami-wallet#662]
The current testnet network nami wallet used is old testnet

Is there a timeline to change the network to Pre-Prod for developers?

**Comment by alessandrokonrad** [comment]
Yes, I am working mostly on the vasil branch right now. Preprod and preview networks are already there: https://github.com/Berry-Pool/nami-wallet/tree/vasil
I'm gonna merge that branch into main as soon as the hard fork occurs. For now if you want to make use of it you have to use the vasil branch version. There is a download link to a full build in the readme on the vasil branch.

**Comment by nmaddp1995** [comment]
@alessandrokonrad Do you know when the hard fork occur. Currently the testnet is really poor
